aeron-test-support/src/main/java/io/aeron/test/cluster/TestCluster.java

```diff
@@ -60,7 +60,9 @@
 {
     private static final int SEGMENT_FILE_LENGTH = 16 * 1024 * 1024;
     private static final long CATALOG_CAPACITY = 128 * 1024;
-    private static final String LOG_CHANNEL = "aeron:udp?term-length=512k";
+//    private static final String LOG_CHANNEL = "aeron:udp?term-length=512k";
+    private static final String LOG_CHANNEL =
+        "aeron:udp?term-length=512k|endpoint=224.20.30.39:24326|interface=localhost";
```

Comment:
Do we always want to test with multicast? Most use will be MDC.

Reply:
I wanted to see some runs in CI with multicast enabled.  I've moved it back but left the configuration commented.  I'll pick up a later task to make this a parameter to our cluster tests.